nrf_modem/doc/fault_handling.rst

```diff
@@ -35,19 +35,19 @@ The application can reinitialize the modem by reinitializing the Modem library t
 .. important::
    If modem traces are enabled, the modem continues to output trace data (a coredump) in the event of a fault, even after signaling the fault to the application.
    To make sure the application is able to retrieve the coredump correctly, the application must not re-initialize the modem until all the outstanding trace data has been processed.
-   The :c:func:`nrf_modem_trace_get()` function will return ``-ENODATA`` when all outstanding trace data has been processed by the application.
+   The :c:func:`nrf_modem_trace_get()` function will return :c:macro:`-NRF_ENODATA` when all outstanding trace data has been processed by the application.
```

Comment:
Does it work to have the `-` minus in the macro name? Those generate links in HTML to the macro definition, how are they working with the minus sign in front?

Reply:
That is a good point..
Maybe do as @MirkoCovizzi suggested in #42, and write
```
return negative :c:macro:`NRF_ENODATA`
```

Reply:
Last changes ended up in the wrong commit, if I'm not mistaken.